src/test/java/io/vertx/core/http/HttpTest.java

```diff
@@ -5048,6 +5048,19 @@ public void testKeepAliveTimeout() throws Exception {
     testKeepAliveTimeout(options, 1);
   }
 
+  @Test
+  public void testKeepAliveTTL() throws Exception {
+    server.requestHandler(req -> {
+      req.response().end();
+    });
+    HttpClientOptions options = createBaseClientOptions()
```

Comment:
I believe we should use longer keepAliveTimeout values here like 3 seconds

Reply:
Sorry didn't understand you here.
I used 300 sec as keepAliveTimeout and 3 sec as keepAliveTTL. So connection should close after 3 sec.

Reply:
ok sorry